apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -36,6 +36,59 @@ namespace Google.Cloud.Spanner.V1
     /// </summary>
     public sealed partial class SessionPool
     {
+        /// <summary>
+        /// This struct contains information required to identify the SessionPool.
+        /// It will be used as key in <see cref="_targetedPools"/>.
+        /// </summary>
+        public readonly struct PartitionKey: IEquatable<PartitionKey>
```

Comment:
I'd suggest moving this into its own file - SessionPool.PartitionKey.cs (to match the other nested types).

Also, please add a space before the colon.

Reply:
Ok Jon will update